build.sbt

```diff
@@ -1,110 +1,99 @@
-import com.typesafe.sbt.SbtScalariform.ScalariformKeys
 import scalariform.formatter.preferences._
 import ReleaseTransformations._
 
 enablePlugins(CrossPerProjectPlugin)
 
 lazy val core = (project in file("core")).
-  settings(settings,
-    tutTargetDirectory := file(".")
-  )
+  enablePlugins(TutPlugin).
+  settings(commonSettings, tutTargetDirectory := file("."))
 
 lazy val docs = (project in file("docs")).
-  settings(settings, publishArtifact := false).
+  enablePlugins(TutPlugin).
+  settings(commonSettings, publishArtifact := false).
   dependsOn(core)
 
-lazy val cats = (project in file("modules/cats")).
-  settings(settings).
-  dependsOn(core)
+def module(proj: Project) = proj.
+  enablePlugins(TutPlugin).
+  dependsOn(core).
+  dependsOn(core % "test->test"). // In order to reuse the scalacheck generators
```

Comment:
Yes, that's the reason why. As I and @melrief discussed before, a more elegant solution would pass by either a `testkit` project with the test helpers, which besides being used internally could be used by other pureconfig users for their own `ConfigConvert` instances, or a private `tests` project with the tests for the core and all modules. Meanwhile, this is the solution we adopted, and every module should have access to those helpers.